<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowPantallaOpcionesUsuario</class>
 <widget class="QMainWindow" name="WindowPantallaOpcionesUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionUsuarios"/>
   <addaction name="actionAlmac_n"/>
   <addaction name="actionOpciones_Usuario"/>
   <addaction name="actionEstad_sticas"/>
   <addaction name="actionLista_de_Compra"/>
   <addaction name="actionProveedores"/>
   <addaction name="actionCambiar_Contrase_a"/>
   <addaction name="actionSalir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArt_culos">
    <property name="title">
     <string>Artículos</string>
    </property>
    <addaction name="actionVer_Art_culos"/>
    <addaction name="actionIntroducir_Compra"/>
    <addaction name="actionSalida_de_Art_culos"/>
    <addaction name="actionCompra_Autom_tica"/>
    <addaction name="actionNuevo_Tipo_de_Producto"/>
    <addaction name="actionTipos_de_art_culos"/>
   </widget>
   <widget class="QMenu" name="menuProveedores">
    <property name="title">
     <string>Proveedores</string>
    </property>
    <addaction name="actionNuevo_Proveedor"/>
    <addaction name="actionArt_culos_de_Proveedor"/>
    <addaction name="actionMostrar_proveedores"/>
   </widget>
   <widget class="QMenu" name="menuUsuarios">
    <property name="title">
     <string>Usuarios</string>
    </property>
    <addaction name="actionA_adir_Nuevo_Usuario"/>
    <addaction name="actionMostrar_usuarios"/>
    <addaction name="actionCambiar_Contrase_a_2"/>
   </widget>
   <widget class="QMenu" name="menuEmpresa">
    <property name="title">
     <string>Empresa</string>
    </property>
    <addaction name="actionModificar_Datos"/>
   </widget>
   <widget class="QMenu" name="menuEstad_sticas">
    <property name="title">
     <string>Estadísticas</string>
    </property>
    <addaction name="actionEstad_sticas_2"/>
    <addaction name="separator"/>
    <addaction name="actionTOP_Art_culos"/>
    <addaction name="actionTOP_proveedores"/>
   </widget>
   <widget class="QMenu" name="menuActividad">
    <property name="title">
     <string>Actividad</string>
    </property>
    <addaction name="actionActividad_Usuario"/>
    <addaction name="actionConexiones"/>
   </widget>
   <widget class="QMenu" name="menuIdioma">
    <property name="title">
     <string>Idioma</string>
    </property>
    <addaction name="actionEspa_ol"/>
    <addaction name="actionIngl_s"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="actionQt_Creator"/>
    <addaction name="actionAcerca_de_GestiStock_Software"/>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionCopia_de_Seguridad_2"/>
    <addaction name="actionRestaurar_Copia_de_Seguridad"/>
   </widget>
   <addaction name="menuActividad"/>
   <addaction name="menuArt_culos"/>
   <addaction name="menuProveedores"/>
   <addaction name="menuEmpresa"/>
   <addaction name="menuUsuarios"/>
   <addaction name="menuEstad_sticas"/>
   <addaction name="menuHerramientas"/>
   <addaction name="menuIdioma"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <action name="actionAlmac_n">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/ark.png</normaloff>:/imagenes/ark.png</iconset>
   </property>
   <property name="text">
    <string>Almacén</string>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/Login Manager.png</normaloff>:/imagenes/Login Manager.png</iconset>
   </property>
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/exit.png</normaloff>:/imagenes/exit.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionEstad_sticas">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/grafica.png</normaloff>:/imagenes/grafica.png</iconset>
   </property>
   <property name="text">
    <string>Gráficas</string>
   </property>
  </action>
  <action name="actionLista_de_Compra">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/kate.png</normaloff>:/imagenes/kate.png</iconset>
   </property>
   <property name="text">
    <string>Lista de compra</string>
   </property>
   <property name="toolTip">
    <string>Lista de compra</string>
   </property>
  </action>
  <action name="actionOpciones_Usuario">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/kuser.png</normaloff>:/imagenes/kuser.png</iconset>
   </property>
   <property name="text">
    <string>Modificar usuario</string>
   </property>
   <property name="toolTip">
    <string>Modificar Usuario</string>
   </property>
  </action>
  <action name="actionProveedores">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/proveedor.png</normaloff>:/imagenes/proveedor.png</iconset>
   </property>
   <property name="text">
    <string>Proveedores</string>
   </property>
   <property name="toolTip">
    <string>Mostrar proveedores</string>
   </property>
  </action>
  <action name="actionCambiar_Contrase_a">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/password.png</normaloff>:/imagenes/password.png</iconset>
   </property>
   <property name="text">
    <string>Cambiar contraseña</string>
   </property>
   <property name="toolTip">
    <string>Cambiar contraseña</string>
   </property>
  </action>
  <action name="actionCopia_de_Seguridad">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/kfloppy.png</normaloff>:/imagenes/kfloppy.png</iconset>
   </property>
   <property name="text">
    <string>Copia de seguridad</string>
   </property>
  </action>
  <action name="actionNuevo_Proveedor">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/add.png</normaloff>:/imagenes/add.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo proveedor</string>
   </property>
  </action>
  <action name="actionVer_Art_culos">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/ark.png</normaloff>:/imagenes/ark.png</iconset>
   </property>
   <property name="text">
    <string>Almacén</string>
   </property>
  </action>
  <action name="actionA_adir_Nuevo_Usuario">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/addUser.png</normaloff>:/imagenes/addUser.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo usuario</string>
   </property>
  </action>
  <action name="actionCambiar_Contrase_a_2">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/password.png</normaloff>:/imagenes/password.png</iconset>
   </property>
   <property name="text">
    <string>Cambiar Contraseña</string>
   </property>
  </action>
  <action name="actionModificar_Datos">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/advancedsettings.png</normaloff>:/imagenes/advancedsettings.png</iconset>
   </property>
   <property name="text">
    <string>Modificar datos</string>
   </property>
  </action>
  <action name="actionActividad_Usuario">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/activity.png</normaloff>:/imagenes/activity.png</iconset>
   </property>
   <property name="text">
    <string>Actividad usuarios</string>
   </property>
  </action>
  <action name="actionCompra_Autom_tica">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/kate.png</normaloff>:/imagenes/kate.png</iconset>
   </property>
   <property name="text">
    <string>Lista de compra</string>
   </property>
  </action>
  <action name="actionCopia_de_Seguridad_2">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/kfloppy.png</normaloff>:/imagenes/kfloppy.png</iconset>
   </property>
   <property name="text">
    <string>Copia de seguridad</string>
   </property>
  </action>
  <action name="actionRestaurar_Copia_de_Seguridad">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/restore.png</normaloff>:/imagenes/restore.png</iconset>
   </property>
   <property name="text">
    <string>Restaurar copia de seguridad</string>
   </property>
  </action>
  <action name="actionArt_culos_de_Proveedor">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/articulo.png</normaloff>:/imagenes/articulo.png</iconset>
   </property>
   <property name="text">
    <string>Artículos de proveedor</string>
   </property>
  </action>
  <action name="actionNuevo_Tipo_de_Producto">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/add.png</normaloff>:/imagenes/add.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo tipo</string>
   </property>
  </action>
  <action name="actionConexiones">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/connect.png</normaloff>:/imagenes/connect.png</iconset>
   </property>
   <property name="text">
    <string>Conexiones</string>
   </property>
  </action>
  <action name="actionEspa_ol">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/Idspain.png</normaloff>:/imagenes/Idspain.png</iconset>
   </property>
   <property name="text">
    <string>Español</string>
   </property>
  </action>
  <action name="actionIngl_s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/IdUnitedKingdom.png</normaloff>:/imagenes/IdUnitedKingdom.png</iconset>
   </property>
   <property name="text">
    <string>Inglés</string>
   </property>
  </action>
  <action name="actionEstad_sticas_2">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/grafica.png</normaloff>:/imagenes/grafica.png</iconset>
   </property>
   <property name="text">
    <string>Gráficas</string>
   </property>
  </action>
  <action name="actionIntroducir_Compra">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/shop add.png</normaloff>:/imagenes/shop add.png</iconset>
   </property>
   <property name="text">
    <string>Compras</string>
   </property>
  </action>
  <action name="actionTOP_proveedores">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/GOLD.png</normaloff>:/imagenes/GOLD.png</iconset>
   </property>
   <property name="text">
    <string>TOP proveedores</string>
   </property>
  </action>
  <action name="actionTOP_Art_culos">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/gold1.png</normaloff>:/imagenes/gold1.png</iconset>
   </property>
   <property name="text">
    <string>TOP Artículos</string>
   </property>
  </action>
  <action name="actionSalida_de_Art_culos">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/shop remove.png</normaloff>:/imagenes/shop remove.png</iconset>
   </property>
   <property name="text">
    <string>Salidas</string>
   </property>
  </action>
  <action name="actionQt_Creator">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/qtcreator.png</normaloff>:/imagenes/qtcreator.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
  <action name="actionTipos_de_art_culos">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/type.png</normaloff>:/imagenes/type.png</iconset>
   </property>
   <property name="text">
    <string>Tipos de artículos</string>
   </property>
  </action>
  <action name="actionAcerca_de_GestiStock_Software">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/logoSolo.png</normaloff>:/imagenes/logoSolo.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de GestiStock Software</string>
   </property>
  </action>
  <action name="actionMostrar_proveedores">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/proveedor.png</normaloff>:/imagenes/proveedor.png</iconset>
   </property>
   <property name="text">
    <string>Proveedores</string>
   </property>
  </action>
  <action name="actionMostrar_usuarios">
   <property name="icon">
    <iconset resource="fuentes.qrc">
     <normaloff>:/imagenes/Login Manager.png</normaloff>:/imagenes/Login Manager.png</iconset>
   </property>
   <property name="text">
    <string>Mostrar usuarios</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="fuentes.qrc"/>
 </resources>
 <connections/>
</ui>
